docker-compose.yml

```diff
@@ -1,7 +1,12 @@
 version: '3'
 services:
   jira:
-    image: atlassian/jira-software:7 # It is recommended to test with version 7. Most, if not all, compatibility issues between Jira Server/Cloud exist in version 7.
-    # image: atlassian/jira-software:8 # Test with version 8 as well when adding new webhook handling logic, or introducing a new Jira API call.
+    container_name: mattermost-jira
+    image: atlassian/jira-software:8 # It is recommended to test with version 8. Most, if not all, compatibility issues between Jira Server/Cloud exist in version 8. 
+    # image: atlassian/jira-software:latest # Alternatively you can test with latest version as well (version 9)
```

Comment:
> It is recommended to test with version 8. Most, if not all, compatibility issues between Jira Server/Cloud exist in version 8

I don't think this new sentence is true. Thankfully we haven't run into issues with Jira 7 lately. We have been trying to be extra cognisant of additional breaking changes in the effort of supporting Jira 9 recently.

Anyway I would say Jira 8 is the most "stable". Jira 9 is a toss-up right now, as more than one issue has come up, but we think it's stable now. Jira 7 remains less stable than Jira 8 due to complications of Jira's API changes.

Reply:
Also I think Jira 9 should be the default now, since that seems to generally be what our customers are using now.

Reply:
We may want to announce that we don't plan to support version 7 anymore, if we are no longer recommending for testing  Jira 7in this file or in docs.